app/views/shared/documents/_preview.html.erb

```diff
@@ -9,12 +9,12 @@
   <div class="card-footer">
     <div class="card-footer__inner d-flex justify-content-around">
       <% actions.each do |action| %>
-        <% if action[:form] %>
+        <% if action.is_a? String %>
```

Comment:
I think the form action is only used in one or two places - maybe we could just pass that here in the same way you're passing the other buttons in, so we can just have `<%= action %>` here. 

Otherwise - we should switch these around - `if action[:form] { ... } else { <%= action %> }` - so that we still have an `else` here

Reply:
switched around